src/dev/i18n/integrate_locale_files.ts

```diff
@@ -206,19 +215,94 @@ async function writeMessages(
   }
 }
 
+function isSpreadableObject(value: unknown) {
+  try {
+    return { ...value }, true;
+  } catch {
+    return false;
+  }
+}
+
+function isEmptyObject(obj: Record<any, any>) {
+  for (const key in obj) {
+    if (obj.hasOwnProperty(key)) {
+      return false; // Found a key, so it is NOT empty
+    }
+  }
+  return true;
```

Comment:
we don't have lodash for these?

Reply:
We are actively trying to phase out lodash.